### ISSUE TITLE
Focus doesn't go to top of the screen/page when COVID Alert menu is opened with VO

### ISSUE THREAD

**Issue by qanitah** [cds-snc/covid-shield-mobile#488]
With VoiceOver on, when the user double taps the menu, the focus is on "How it protects your privacy". It should be on top of the page. I tried it multiple times and it always goes to this which is the last item on the page. 

Screenshot has From and To pages.

iOS 13.5.1 on iPhone 7

![image](https://user-images.githubusercontent.com/14280062/86809115-82edf580-c049-11ea-99d0-71386705a8bc.png)

**Comment by JuliannaR** [comment]
@henrytao-me the focus should be on Diagnosed with COVID-19. When the sheet is collapsed focus should also be to the header in the page content.

**Comment by henrytao-me** [comment]
@qanitah @JuliannaR PR is up https://github.com/cds-snc/covid-shield-mobile/pull/557. 

Questions, what element should we focus on when: 
- App is opened.
- BottomSheet is opened. Will we force VoiceOver / TalkBack to focus every time BottomSheet is opened? 
- BottomSheet is closed.

**Comment by JuliannaR** [comment]
@henrytao-me when you close the shield focus should land on the header to give context to the user. Otherwise they start a various points in the content being displayed and its very disorienting.

**Comment by JuliannaR** [comment]
@qanitah I have combined the focus issues here https://github.com/cds-snc/covid-shield-mobile/issues/563
